engine/render/src/render/font/font_glyphbank.cpp

```diff
@@ -0,0 +1,189 @@
+// Copyright 2020-2025 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+//
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+//
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+#include <stdlib.h>
+#include <font/font.h>
+
+
+#include <render/font_ddf.h>
+
+#include "font_glyphbank.h"
+
+struct GlyphBankFont
+{
+    Font m_Base;
+    dmRenderDDF::GlyphBank* m_GlyphBank;
+};
+
+static dmRenderDDF::GlyphBank* ToGlyphBank(HFont hfont)
+{
+    return ((GlyphBankFont*)hfont)->m_GlyphBank;
+}
+
+
+static void FontDestroyGB(HFont hfont)
+{
+    GlyphBankFont* font = (GlyphBankFont*)hfont;
+    // The actual data comes from the resource system, so no need to free that
+    free((void*)font);
+}
+
+static uint32_t GetResourceSizeGB(HFont hfont)
```

Comment:
`GB` can be confusing here because `SizeGB` reads as `resource size in Gigabytes` rather than `size of Glyph bank`.